servo-build-dependencies/init.sls

```diff
@@ -1,7 +1,16 @@
+{% from 'common/map.jinja' import common %}
+
 include:
   - python
 
 servo-dependencies:
+  cmd.run:
+    - name: |
+        curl https://sh.rustup.rs -sSf |
+        sh -s -- --default-toolchain none -y
+    - runas: servo
+    - creates:
+      - {{ common.servo_home }}/.rustup
```

Comment:
I think it also creates `.cargo`, and creates-or-modifies `.profile` to add `.cargo/bin` to `$PATH`.